examples/sensor/sensor/sensor.py

```diff
@@ -0,0 +1,49 @@
+# %% [markdown]
+# # Sensoring File
+#
+# This example shows how to use the `SensorTask` to execute a query in Sensor.
+#
+# To begin, import the required libraries.
+
+# %%
+from flytekit import task, workflow
+from flytekit.sensor.file_sensor import FileSensor
+
+# %% [markdown]
+# Create a Sensor task.
+#
+# The sensor will find the file at the given path.
```

Comment:
```suggestion
# The sensor will search for the file at the specified path. If the file exists, it will return a succeed status. Otherwise, the sensor will continue running until the file is added to the directory.
```